<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormAlterarUsuario</class>
 <widget class="QMainWindow" name="FormAlterarUsuario">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>489</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Alterar Usuário</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../img/cadastrar usuario 2.png</normaloff>../../img/cadastrar usuario 2.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Alterar Usuário</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>170</y>
      <width>168</width>
      <height>35</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="btnSalvar">
       <property name="text">
        <string>Cadastrar</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../img/salvar usuario.png</normaloff>../../img/salvar usuario.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="btnLimpar">
       <property name="text">
        <string>Limpar</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../img/borracha.png</normaloff>../../img/borracha.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>70</y>
      <width>311</width>
      <height>86</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="labelUsuario">
       <property name="text">
        <string>Usuário:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="editUsuario">
       <property name="inputMask">
        <string/>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelSenha">
       <property name="text">
        <string>Senha:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="editSenha">
       <property name="inputMask">
        <string/>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>editUsuario</tabstop>
  <tabstop>btnSalvar</tabstop>
  <tabstop>btnLimpar</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btnLimpar</sender>
   <signal>clicked()</signal>
   <receiver>editUsuario</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>264</x>
     <y>156</y>
    </hint>
    <hint type="destinationlabel">
     <x>259</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
